parsers/DO.py

```diff
@@ -65,13 +65,13 @@
                  u'PUNTA CATALINA 1': 'coal',
                  u'PUNTA CATALINA 2': 'coal',
                  u'QUISQUEYA 1': 'gas',
-                 u'QUISQUEYA 2': 'gas',
+                 u'QUISQUEYA 2': 'oil',
                  u'QUISQUEYA 1 SAN PEDRO': 'oil',
                  u'RIO SAN JUAN': 'oil',
                  u'SAN FELIPE': 'oil',
                  u'SAN FELIPE CC': 'gas',
                  u'SAN FELIPE VAP': 'oil',
-                 u'SAN LORENZO 1': 'gas',
+                 u'SAN LORENZO 1': 'oil',
```

Comment:
Could you explain this change? I found this which suggests to me that it is a gas plant https://www.power-technology.com/projects/sanlorenzo/

Reply:
Looks like that was a mistake, sorry about that.